framework/docs/source/how-to-migrate-from-openfl.rst

```diff
@@ -0,0 +1,537 @@
+:og:description: Migrate your OpenFL workloads to Flower in this step-by-step tutorial
+.. meta::
+    :description: Migrate your OpenFL workloads to Flower in this step-by-step tutorial
+
+.. _how-to-migrate-from-openfl:
+
+.. |message_link| replace:: ``Message``
+
+.. _message_link: ref-api/flwr.app.Message.html
+
+.. |arrayrecord_link| replace:: ``ArrayRecord``
+
+.. _arrayrecord_link: ref-api/flwr.app.ArrayRecord.html
+
+.. |context_link| replace:: ``Context``
+
+.. _context_link: ref-api/flwr.app.Context.html
+
+.. |clientapp_link| replace:: ``ClientApp``
+
+.. _clientapp_link: ref-api/flwr.clientapp.ClientApp.html
+
+.. |fedavg_link| replace:: ``FedAvg``
+
+.. _fedavg_link: ref-api/flwr.serverapp.strategy.FedAvg.html
+
+.. |serverapp_link| replace:: ``ServerApp``
+
+.. _serverapp_link: ref-api/flwr.serverapp.ServerApp.html
+
+.. |strategy_start_link| replace:: ``start``
+
+.. _strategy_start_link: ref-api/flwr.serverapp.strategy.Strategy.html#flwr.serverapp.strategy.Strategy.start
+
+.. |strategy_link| replace:: ``Strategy``
+
+.. _strategy_link: ref-api/flwr.serverapp.strategy.Strategy.html
+
+.. |result_link| replace:: ``Result``
+
+.. _result_link: ref-api/flwr.serverapp.strategy.Result.html
+
+OpenFL Migration Guide
+======================
+
+It was `recently announced <https://github.com/securefederatedai/openfederatedlearning>`_ that the The Open Federated Learning project 
```

Comment:
Note for reviewers - the OpenFL readme will be updated with the announcement once this page goes live